examples/svelte-multi-providers/.env.local.sample

```
@@ -0,0 +1,23 @@
+# VITE_OIDC_ISSUER_URI=https://cloud-iam.oidc-spa.dev/realms/oidc-spa
+# VITE_OIDC_CLIENT_ID=example-react-router
+
+# ======= Microsoft Entra ID =======
+VITE_OIDC_ISSUER_URI_MICROSOFT=https://login.microsoftonline.com/71a0a621-363a-4182-8209-86364aa6de03/v2.0
+VITE_OIDC_CLIENT_ID_MICROSOFT=3c8f72b4-4dd9-4aa2-a01d-43349ba90394
+VITE_OIDC_SCOPE_MICROSOFT="profile api://example-multi-providers/access_as_user"
+
+# ======= Google OAuth =============
+VITE_OIDC_ISSUER_URI_GOOGLE=https://accounts.google.com
+VITE_OIDC_CLIENT_ID_GOOGLE=928024164279-ifjvmsffi64slkk81h3gmoh7p03ev68k.apps.googleusercontent.com
+VITE_OIDC_CLIENT_SECRET_GOOGLE=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
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The VITE_OIDC_CLIENT_SECRET_GOOGLE key should go before the VITE_OIDC_ISSUER_URI_GOOGLE key

(UnorderedKey)